### DISCUSSION TITLE
Connector request of bank transactions - wonderful.co.uk

### DISCUSSION THREAD

**Discussion by arunura** [juspay/hyperswitch#5040] [Ideas / Feature Requests]
[Wonderful](https://wonderful.co.uk/#pricing) is one of the cheapest (if not the cheapest) processor for Local Bank transactions in the UK. The user experience is fantastic, and they have integration with all the major banks in the UK.

A connector for wonderful will allow nonprofits, to use hyperswitch for fundraising through donations at absolutely **ZERO** fee. For smaller nonprofits run by volunteers the payment costs are significant erosion in donations. It's not straight forward to use wonderful at the moment, and having this connector will open up ZERO fee fundraising to a **number of nonprofits** in the UK.

**Comment by gorakhnathy7** [comment]
Hey @arunura,
Thanks for elaborating the use case, it certainly seems like a pain point. 

We will check the feasibility of Integrating Wonderful Connector and keep you posted with an update.
